tabcmd/commands/datasources_and_workbooks/export_command.py

```diff
@@ -57,16 +59,17 @@ def define_args(export_parser):
             help="Set the page size of the exported PDF",
         )
 
-        group.add_argument(
-            "--width", default=800, help="[Not Yet Implemented] Set the width of the image in pixels. Default is 800 px"
-        )
+        group.add_argument("--width", default=800, help=_("export.options.width"))
         group.add_argument("--filename", "-f", help="filename to store the exported data")
         group.add_argument("--height", default=600, help=_("export.options.height"))
         group.add_argument(
             "--filter",
             metavar="COLUMN=VALUE",
             help="Data filter to apply to the view",
         )
+        group.add_argument(
+            "--resolution", choices=[imageresolution.High], type=str.lower, help=_("export.options.resolution")
```

Comment:
Might be better to keep the default to high-res, so people don't suddenly get different results from their existing processes